_posts/2024-07-11-Optimize-storage-and-performance-using-MatchOnlyText-field.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title:  "Optimize storage and performance with the MatchOnlyText field in OpenSearch"
+authors:
+  - rishabhmaurya
+  - macrakis
+date: 2024-07-11
+categories:
+  - technical-posts
+meta_keywords: match_only_text field, text field, text query performance, storage optimization
+meta_description: match_only_text field could a replacement of text field, when positions are not required to be indexed, providing better performance and reduced cost.
```

Comment:
Meta updates: 

meta_keywords: match_only_text field, storage optimization, full-text search OpenSearch, text query performance

meta_description: Explore how the match_only_text field can help you work with large datasets in OpenSearch while making it possible to optimize storage and performance at a reduced cost.